tidb-resource-control.md

```diff
@@ -0,0 +1,118 @@
+---
+title: 使用 Resource Control 实现资源隔离
+summary: 介绍如何通过 Resource Control 来实现应用资源消耗的控制和有效调度
+---
+
+# 使用 Resource Control 实现资源隔离
+
+Resource control 特性允许集群管理员通过定义资源组(Resource Group)，资源组可以限定读写的配额；将用户绑定到某个资源组，TiDB 层会根据用户所绑定资源组设定的读写配额对用户的读写请求做流控；同时，TiKV 层会使用读写配额映射的优先级来对请求做调度。通过流控加上调度两层控制，你可以实现应用的资源隔离，满足服务质量（QoS）要求。
+
+Resource control 特别适用于在大集群内部实现多用户应用的资源隔离。
+
+> **警告:**
+>
+> Resource Control 是 TiDB 在 v6.6.0 中引入的实验特性，其语法或者行为表现在 GA 前可能会发生变化。如果你知晓潜在的风险，可通过执行 TiDB SQL 语句`SET GLOBAL tidb_enable_resource_control = 'ON'`， 同时设置 TiKV 配置参数 `resource_control.enabled` 为 'true' 来开启该实验特性。
+
+## 新参数
+
+Resource Control 分别向 TiDB 和 TiKV 引入新的参数或变量进行全局控制：
+* TiDB: 通过全局变量 `tidb_enable_resource_control` 控制是否打开资源组流控。
+* TiKV: 通过参数配置 `resource_control.enabled` 控制是否使用基于资源组配额的请求调度。此参数暂时不支持动态修改，修改后需要重启 TiKV 实例生效。
+
+* 这2个参数的组合效果见下表：
+
++-----------------------------------+--------------------------------------+--------------------------------------+
+|                                   | `tidb_enable_resource_control`= ON   | `tidb_enable_resource_control`= OFF  |
++-----------------------------------+--------------------------------------+--------------------------------------+
+| `resource_control.enabled`= true  |          流控和调度（推荐组合）          |               无效配置                ｜
++-----------------------------------+--------------------------------------+--------------------------------------+
+| `resource_control.enabled`= false |                 仅流控                |               特性被关闭              ｜
++-----------------------------------+-----------------------------------------------------------------------------+
```

Comment:
@hfxsd please help fix the table format.
<img width="1033" alt="image" src="https://user-images.githubusercontent.com/97348524/213248557-a3fea7f9-269b-4c27-bd28-f6914dfaafd0.png">

Reply:
done